loopin.go

```diff
@@ -18,7 +18,7 @@ import (
 	"github.com/lightninglabs/loop/loopdb"
 	"github.com/lightninglabs/loop/swap"
 	"github.com/lightningnetwork/lnd/chainntnfs"
-	"github.com/lightningnetwork/lnd/channeldb"
+	invpkg "github.com/lightningnetwork/lnd/invoices"
```

Comment:
was there a conflict with `invoices`?

Reply:
It's just a convention that we seemed to have picked up in the LND codebase so I was just meant to conform to that.